DigitalLearningSolutions.Data.Migrations/202107221605_ChangeEmailMaxLengthOnAdminUsers.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202107221605)]
+    public class ChangeEmailMaxLengthOnAdminUsers : Migration
+    {
+        public override void Up()
+        {
+            Delete.Index("IX_AdminUsers_Email").OnTable("AdminUsers");
+            Alter.Column("Email").OnTable("AdminUsers").AsString(255);
+            Create.Index("IX_AdminUsers_Email").OnTable("AdminUsers").OnColumn("Email").Ascending().WithOptions()
+                .Unique().WithOptions().NonClustered();
```

Comment:
Do we need "WithOptions" twice?

Reply:
I first had it only before Unique, but I was getting a squiggly on Nonclustered saying `CS1061 'ICreateIndexOnColumnSyntax' does not contain a definition for 'NonClustered' and no accessible extension method 'NonClustered' accepting a first argument of type 'ICreateIndexOnColumnSyntax' could be found (are you missing a using directive or an assembly reference?)`

If I remove WithOptions before Unique the same thing happens on Unique, so I'm assuming this is just the syntax? But I don't know, I've never written migrations in c sharp before 🙃